### PR TITLE
Phases Status and Price Data Fix

### PR DESCRIPTION
**Pull request by suppermancool**
Phases Status and Price Data Fix

### CODE REVIEW

File: src/projects/detail/components/ProjectStage.jsx [hunk 1]
```diff
@@ -35,30 +35,30 @@ function formatPhaseCardAttr(phase, productTemplates, feed) {
...
   const endDate = phase.endDate && moment(phase.endDate)
   const duration = startDate && endDate
-    ? moment.duration(endDate.diff(startDate)).days() + ' days'
+    ? (endDate.diff(startDate, 'days') + 1) + ' days'
     : '0 days'
   let startEndDates = startDate ? `${startDate.format('MMM D')}` : ''
   startEndDates += startDate && endDate ? `–${endDate.format('MMM D')}` : ''
 
-  const actualPrice = product.actualPrice
+  const actualPrice = product.actualPrice || 0
```
Comment (RishiRajSahu) [inline src/projects/detail/components/ProjectStage.jsx]: @suppermancool are we using this actual price somewhere ?

Comment (suppermancool) [reply src/projects/detail/components/ProjectStage.jsx]: yeah the old code use it to calculate the paidStatus. But i think my change break the old logic, just reverted the code in b45bf24
https://gyazo.com/e94a790ca58e5f41214382b01efe9d0a

File: src/projects/actions/project.js [hunk 2]
```diff
@@ -47,6 +47,35 @@ function populatePhasesProducts(result) {
...
 }
 
+/**
+ * Populate each phase with `directProject` property containing direct project
+ *
+ * @param {Array} phases list of phases
+ *
+ * @return {Promise} modified array of phases with `directProject` property
+ */
+function populatePhasesDirectProject(phases) {
```
[the remark this answers is not in the document]
Comment (RishiRajSahu) [reply src/projects/actions/project.js]: @suppermancool

File: src/projects/detail/components/ProjectPlanProgress.jsx [hunk 3]
```diff
@@ -16,13 +17,23 @@ import ProjectProgress from './ProjectProgress'
  *
  * @return {Object} ProjectProgress props
  */
-function formatProjectProgressProps(project) {
-  const actualDuration = _.get(project, 'duration.actualDuration', 0)
-  const projectedDuration = _.get(project, 'duration.projectedDuration', 0)
+function formatProjectProgressProps(project, phases) {
+  const actualDuration =  _.sumBy(phases, 'directProject.actualDuration')
```
Comment (RishiRajSahu) [inline src/projects/detail/components/ProjectPlanProgress.jsx]: @suppermancool we don't have to sum duration of same direct project again, it should be counted once.

File: src/projects/actions/project.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (suppermancool) [reply src/projects/actions/project.js]: done in 2aaa5c0

File: src/projects/detail/components/ProjectPlanProgress.jsx (hunk 3 above)
Comment (suppermancool) [reply src/projects/detail/components/ProjectPlanProgress.jsx]: done in 2aaa5c0

**Pull request merged**